packages/cysync-core-constants/src/i18n/lang/en.json

```diff
@@ -1040,6 +1040,16 @@
           "heading": "Something went wrong",
           "subtext": "Try again and if the problem persists, contact Cypherock support for assistance"
         }
+      },
+      "xrp": {
+        "XRP_0000": {
+          "heading": "Some unknown error occured",
+          "subtext": "Try again and if the problem persists, contact Cypherock support for assistance"
+        },
+        "XRP_0001": {
+          "heading": "Transaction Failed",
+          "subtext": "Your transaction may have failed. Please wait a moment then check the transaction history before trying again and if the problem persists, contact Cypherock support for assistance"
+        }
```

Comment:
Do not add xrp error strings for now, we will have to add this when proper xrp server error mapping is created

Reply:
removed al xrp error related strings